lib/generators/noticed/model_generator.rb

```diff
@@ -42,7 +42,11 @@ def model_path
       end
 
       def params_column
-        case ActiveRecord::Base.configurations.configs_for(spec_name: "primary").config["adapter"]
+        case if ActiveRecord::Base.respond_to?(:connection_db_config)
+               ActiveRecord::Base.connection_db_config.adapter
+             else
+               ActiveRecord::Base.connection_config[:adapter]
+             end
```

Comment:
Can we move this to a method like `def adapter` so we don't have an if inside the case?

Reply:
You bet!